tsconfig.json

```diff
@@ -6,7 +6,10 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "noImplicitAny": true,
-    "strictNullChecks": true
+    "strictNullChecks": true,
+    "types": [
+      "node"
+    ]
```

Comment:
Well, this is annoying. Basically, our inclusion of `chrome-launcher` adds `@types/core-js` to our dependencies, which TypeScript then gets fussy over because I guess it's an ancient version of that library or something.  In any case, we're not actually type-checking any of our code that uses `chrome-launcher` or `core-js`.  There's no way to black-list packages under `@types` in TypeScript, so instead we'll have to take the whitelisting approach and explicitly list every `@types` package we want to enable here. Blerg.